study.md

```diff
@@ -34,7 +34,7 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+Array and Enumerable both share any? and map.  For both of these, there is no discernible difference between the methods listed on array and enumerable.  I think they are defined on both array and enumerable because the one defined on enumerable can be used for other collections besides array so it helps to be able to reference them from both objects when someone needs to look them up.
```

Comment:
This does not answer the question.

`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.